content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -281,6 +281,78 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Kubernetes' implementation of dual-stack networking is in flux. You may well find that the design is changed in future releases.{{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simulatenously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
+
+   * Dual stack Pod networking (multi-IP Pod)
+   * Kubenet multi address familiy support (IPv4 and IPv6)
+   * Egress Pod Internet routing via IPv4/IPv6 addresses
+
+### Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual stack Kubernetes clusters:
+
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, the following flags must added to these cluster componenets:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+
+### Validate IPv4/IPv6 dual stack
+
+Validate IPv4/IPv6 Pod CIDRs have been assigned on a dual stack enabled node by running the following command. There should be a single IPv4 and IPv6 CIDR allocated (replace node name with a valid node from the cluster. In this example the node name is k8s-linuxpool1-34450317-0):
+
+```shell
+kubectl get nodes k8s-linuxpool1-34450317-0 -o go-template --template='{{range .spec.podCIDRs}}{{printf "%s\n" .}}{{end}}'
+10.244.1.0/24
+a00:100::/24
```

Comment:
```suggestion
fd00:db8::/24
```
This is a local (RFC 4193) IPv6 address block.

If the implementation already worked OK with /32 blocks, I would re `2001:DB8::/32` because `2001:DB8::/32` is reserved for documentation use.